api/app/src/main/kotlin/packit/controllers/PacketController.kt

```diff
@@ -41,15 +41,15 @@ class PacketController(private val packetService: PacketService)
         )
     }
 
-    @GetMapping("/packetGroupSummary")
-    fun getPacketGroupSummary(
+    @GetMapping("/packetGroupSummaries")
```

Comment:
This is one of a few renamings i did for increased precision. The endpoint returns a summary for each packet group (or, if a name to filter on is provided, a summary for each _matching_ packet group).